front/app/containers/IdeasNewPage/index.tsx

```diff
@@ -57,15 +58,15 @@ const IdeasNewPage = () => {
     return <PageNotFound />;
   }
 
-  if (!phases) {
+  if (!phases || !phaseId) {
     return null;
   }
 
   const currentPhase = getCurrentPhase(phases.data);
   const participationMethod = getParticipationMethod(
     project.data,
-    phases?.data,
-    phase_id
+    phases.data,
+    phaseId
```

Comment:
This is more or less the same as before. By using `phaseId`, we use either `phase_id` from search params or the value returned by `getCurrentPhase` is used. The same fallback (calling `getCurrentPhase`) happens in `getParticipationMethod`.